fineract-provider/src/main/java/org/apache/fineract/cob/loan/ApplyLoanLockTasklet.java

```diff
@@ -65,7 +65,7 @@ public RepeatStatus execute(@NotNull StepContribution contribution, @NotNull Chu
             loanIds = Collections.emptyList();
         } else {
             loanIds = new ArrayList<>(loanRepository.findAllNonClosedLoansBehindOrNullByMinAndMaxLoanId(loanCOBParameter.getMinLoanId(),
-                    loanCOBParameter.getMaxLoanId(), ThreadLocalContextUtil.getBusinessDateByType(BusinessDateType.COB_DATE)));
+                    loanCOBParameter.getMaxLoanId(), ThreadLocalContextUtil.getBusinessDateByType(BusinessDateType.COB_DATE).minusDays(1)));
```

Comment:
why -1 day?

Reply:
moved the `NUMBER_OF_DAYS_BEHIND` to `LoanCOBConstant` and use it here as well